XCUITests/SmokeXCUITests.xctestplan

```diff
@@ -9,7 +9,7 @@
     }
   ],
   "defaultOptions" : {
-
+    "codeCoverage" : false
```

Comment:
@nbhasin2 is it fine to modify this setting? I read a post and with that the tests are launched sucessfully on Birise